data/maps/SP_Scenarios/Along_The_River.wmf/scripting/init.lua

```diff
@@ -0,0 +1,385 @@
+      p1:hide_fields(port:region(6))
+      local pirate_fortress = map:get_field(55, 60)
+      if pirate_fortress.immovable and pirate_fortress.owner == p5 then
+         -- Soldier level must match with level in initial_conditions.
+         pirate_fortress.immovable:set_soldiers({0,2,0,2}, 8)
+      end
+      while not immovable_has_name(port.immovable, "barbarians_port") do sleep(2039) end
+      o_build_port.done = true
+      map:get_field(30,143).terd = "summer_mountain2"
+      map:get_field(29,143).terr = "summer_mountain3"
+   end)
+
+   -- Conquer pirate barbarians_sentry and build warehouse.
+   run(function()
+      local pirate_sentry = map:get_field(59, 89)
+      while not p1:seen_field(pirate_sentry) do
+         sleep(2011)
+         -- Ai will dismantle sentry cause it can't be connected to a WH so check and rebuild
+         local sentry_immovable = pirate_sentry.immovable
+         if not sentry_immovable or sentry_immovable.descr.name ~= "barbarians_sentry" then
+            prefilled_buildings(p5, {"barbarians_sentry", 59, 89, soldiers = {[{0,2,0,2}] = 2 }})
+         end
+      end
+      scroll_to_field(pirate_sentry)
+      campaign_message_box(briefing_conquer_pirate_sentry)
+      local o_conquer_pirate_sentry = add_campaign_objective(obj_conquer_pirate_sentry)
+
+      while pirate_sentry.owner == p5 do sleep(997) end
+      p1:place_flag(map:get_field(53, 87), true)
+      map:get_field(53, 87).immovable:remove()
+      p1:reveal_fields(warehouse:region(6))
+      scroll_to_field(warehouse)
+      campaign_message_box(briefing_found_pirate_treasure)
+      o_build_warehouse = add_campaign_objective(obj_build_warehouse)
+      o_conquer_pirate_sentry.done = true
+      run(function()
+         sleep(30000)
+         p1:hide_fields(warehouse:region(6))
+      end)
+      transport_wares(p1, hq, "gold_ore", 25, true, "tribes/buildings/warehouses/barbarians/headquarters_interim/idle_00.png")
+      while not immovable_has_name(warehouse.immovable, "barbarians_warehouse") do sleep(2053) end
+      o_build_warehouse.done = true
+   end)
+
+   while not (obj_done(o_build_warehouse) and obj_done(o_build_port)) do sleep(2063) end
+
+   -- =======================================================================
+   -- Bring shovels and remove the swamps.
+   -- =======================================================================
+
+   -- Wait for shovels in warehouse and hand them (done in "wares_collected_at_field" function)out.
+   run(function()
+      scroll_to_field(warehouse)
+      campaign_message_box(briefing_bring_shovels_1)
+      o_bring_shovels_1 = add_campaign_objective(obj_bring_shovels_1)
+      while wares_collected_at_field(warehouse, "shovel", 4) == false do sleep(2027) end
+      send_to_inbox(p1, _"Shovels", li_image("tribes/buildings/warehouses/barbarians/warehouse/idle_1.png", _"Four shovels for removing the swamps have been collected at the warehouse."), { field = warehouse, popup = true, })
```

Comment:
I suspect both occurrences of this line need to be surrounded with a push/pop _textdomain call